lib/fork.js

```diff
@@ -18,13 +18,29 @@ module.exports = function (args) {
 
 	var ps = childProcess.fork(babel, args, options);
 
+	var killed = false;
+	var killedPromise = new Promise(function (resolve) {
```

Comment:
I don't really understand the need for `killedPromise`. Why don't we listen to `exit` event inside return Promise? https://github.com/jamestalmage/ava/blob/jt-kill-forked-processes/lib/fork.js#L36